bundle/compliance/cis_aws/rules/cis_1_13/rule.rego

```diff
@@ -0,0 +1,16 @@
+package compliance.cis_aws.rules.cis_1_13
+
+import data.compliance.lib.common
+import data.compliance.policy.aws_iam.data_adapter
+
+# Ensure that there is only a single active access key per user.
+finding = result {
+	# filter
+	data_adapter.is_iam_user
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(count(data_adapter.active_access_keys) < 2),
+		{"IAM User:": data_adapter.iam_user},
+	)
```

Comment:
how does this check address the audit described in the rule?

> Check the `Status` property value for each key returned to determine each keys current state. If the `Status` property value for more than one IAM access key is set to `Active`, the user access configuration does not adhere to this recommendation, refer to the remediation below

Reply:
The `active_access_keys` set contains only the activated keys, in case this set holds more than a single key - we should fail.

Reply:
That makes sense, in any case, let's keep the convention of having the audit in a separate file, in order to make it reusable :pray: